RealEstate/apps/core/views.py

```diff
@@ -45,3 +46,83 @@ def get(self, request, *args, **kwargs):
         couple = Couple.objects.filter(homebuyer__user=request.user)
         house = House.objects.filter(couple=couple)
         return render(request, 'core/homebuyerHome.html', {'couple': couple, 'house': house})
+
+
+class EvalView(BaseView):
+    """
+    View for the Home Evaluation Page. Currently, this page is decoupled
+    from the rest of the app and uses static elements in the database.
+    """
+    def get(self, request, *args, **kwargs):
+        homebuyer = Homebuyer.objects.filter(user_id=request.user.id)
+        couple = Couple.objects.filter(homebuyer__user=request.user)
+        categories = Category.objects.filter(couple=couple)
+
+        # A specific house should be passed into the request, changing next two lines.
+        house = House.objects.filter(couple=couple)
+        grades = Grade.objects.filter(house=house.first(), homebuyer=homebuyer)
+
+        # Merging grades and categories to provide object with both information.
+        # Data Structure: [(cat1, score1), (cat2, score2), ...]
+        graded = []
+        for category in categories:
+            missing = True
+            for grade in grades:
+                if grade.category.id is category.id:
+                    graded.append((category, grade.score))
+                    missing = False
+                    break
+            if missing:
+                graded.append((category, None))
+                
+        class ContactForm(forms.Form):
+            def __init__(self, *args, **kwargs):
+                super(ContactForm, self).__init__(*args, **kwargs)
+                for c, s in graded:
+                    self.fields[c.summary] = forms.CharField(initial="3" if None else s, widget=forms.HiddenInput())
```

Comment:
I agree that things would ideally be async, however my main concern is the time considerations since we are already a few weeks into the second term.  So I think the highest priority should be something that works, then we can revisit things to improve them.  The one merge-blocking issue for this pull request is the URLconf, which should include the house ID as described in a separate comment.